diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/FileSizeWarnTest.kt

```diff
@@ -75,4 +85,40 @@ class FileSizeWarnTest {
                         "${Warnings.FILE_IS_TOO_LONG.warnText()} $size", false),
                 rulesConfigList = rulesConfigListLarge)
     }
+
+    @Test
+    fun `configuration is empty`(){
+        val path = javaClass.classLoader.getResource("test/paragraph3/src/main/FileSizeLarger.kt")
+        val file = File(path!!.file)
+        lintMethod(FileSize(), file.absolutePath, file.readText(), rulesConfigList = rulesConfigListEmpty)
```

Comment:
When configuration is empty, you use default value, which is 2000 for lines count. Please change this test name to better represent that check is not switched off but rather doesn't detect small files.